Theme Scrollbars

jfgordon2 opened the pull request:
<!--
What GitHub Desktop issue does this PR address? (for example, #1234)
-->

Closes #236

## Description

Updated scrollbars to match windows webkit styling for both light and dark mode

### Screenshots

![image](https://user-images.githubusercontent.com/55799997/77877405-4c6b3800-721b-11ea-8066-b185118d51de.png)

## Release notes

Notes: Matched webkit-scrollbar styling from windows to linux for consistency and to fix scrollbars in dark mode

jfgordon2 committed:
Theme Scrollbars

Updated scrollbars to match windows webkit styling for both light and dark mode

jfgordon2 committed:
Update _scroll.scss

Separating linux from win32 reference, in case we want to adjust for the future and not impact the other

jfgordon2 commented:
FYI - there's an upstream issue where the scrollbars very hard to see on the dark theme.  I posted [PR 9416](https://github.com/desktop/desktop/pull/9416) to adjust the coloring.

shiftkey commented:
@jfgordon2 if that gets approved and merged you can pull those changes into this branch so it's available for the next Linux release

jfgordon2 commented:
> @jfgordon2 if that gets approved and merged you can pull those changes into this branch so it's available for the next Linux release

You got it :+1:

shiftkey commented:
@jfgordon2 looks like your PR was merged - congratulations! Feel free to pull those changes into this branch, and I think this is good to land aside from the dependency on the mixins that are planned to be removed in #237

jfgordon2 committed:
Updated Scroll Color and Docs

Adding color change to dark theme scroll bars merged upstream, and updating documentation related to editor and shell updates

jfgordon2 commented:
@shiftkey Thank you!  It's now updated

shiftkey reviewed (approved):
I'm going to merge this in now and address #237 later on. Thanks for the quick PR on this @jfgordon2, as well as upstreaming the tweak to the colours!
